src/mnode/ticket-processor.cpp

```diff
@@ -645,12 +645,23 @@ std::string CPastelTicketProcessor::ListFilterActTickets(const short filter) con
             return true;
         });
 }
-// 1 - available; 2 - unavailable; 3 - expired; 4 - sold
-std::string CPastelTicketProcessor::ListFilterSellTickets(const short filter) const
+// 0 - all, 1 - available; 2 - unavailable; 3 - expired; 4 - sold
+std::string CPastelTicketProcessor::ListFilterSellTickets(const short filter, const std::string& pastelID) const
 {
+    bool checkConfirmation = true;
```

Comment:
probable just write:
const bool checkConfirmation{filter > 0};
and remove
"else
checkConfirmation = false;"

Reply:
that will change the logic here :)

Reply:
It will not change logic ) we have checkConfirmation false only for filter == 0. It will work correct if
    
    const bool checkConfirmation{filter > 0}; // do not check confirmation for ALL
    if (filter == 0 && pastelID.empty()) {
      return ListTickets<CArtSellTicket>(); // get all
    }
    return filterTickets<CArtSellTicket>(...
    
    It is not error, just less code

Reply:
I see, will fix